### DISCUSSION TITLE
Add deathmessage toggle option for players so they can hide/show it when they please

### DISCUSSION THREAD

**Discussion by SenorFlamingo** [Zrips/CMI#8124] [Feature Requests]
I think this could be a nice addition to the plugin, because this would give the player the freedom to choose whether they want to see any deathmessages in their chat or not.

**Comment by CloudeLecaw** [comment]
Might it could be added as a variable in the /cmi options (with a default setting in config players options of curse)

For the config.yml player Options:
`showDeathMeassages: true`

And the command variable showDeathMessages could look like this:
`/cmi options (playerName) showdeathmessages (enable/disable/toggle/status) (-s)`

permission:
`cmi.command.options.showdeathmessages`

Placeholder:
`%cmi_user_toggle_showdeathmessages%` and  `%cmi_user_togglename_showdeathmessages%`
